server/src/settings.php

```diff
@@ -10,6 +10,7 @@
 		'determineRouteBeforeAppMiddleware' => true,
 		'db' => [
 			'host' => getenv( 'DB_HOST' ),
+			'cluster' => getenv( 'DB_CLUSTER' ),
```

Comment:
Don't we need to add `DB_CLUSTER` to .env.dist?

Reply:
Not necessarily... I mean we aren't expecting it to be set by anyone other than toolforge. I could add it and comment it out. what do you think?

Reply:
If we are not around and someone needs to re-deploy this they woudln't know they need it so I'd mention it in the readme and leave it empty on .env.dist